rules/public/malware/macos/malware_macos_proton_rat_generic.yara

```diff
@@ -0,0 +1,27 @@
+private rule MachO
+{
+    meta:
+        description = "Mach-O binaries"
+    condition:
+        uint32(0) == 0xfeedface or uint32(0) == 0xcefaedfe or uint32(0) == 0xfeedfacf or uint32(0) == 0xcffaedfe or uint32(0) == 0xcafebabe or uint32(0) == 0xbebafeca
+}
```

Comment:
The `MachO` rule should be moved into its own file and included by each individual rules file.

`macho.yara`:  `private rule MachO { ... }`

`malware_macos_proton_rat_generic.yara`:
```
include "../macho.yara"

....
```

See http://yara.readthedocs.io/en/v3.6.3/writingrules.html#including-files